js/app/controllers/editorcontroller.js

```diff
@@ -53,15 +53,16 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 			{title: t('calendar', 'Details'), value: 0},
 			{title: t('calendar', 'Attendees'), value: 1},
 			{title: t('calendar', 'Reminders'), value: 2},
-			{title: t('calendar', 'Repeating'), value: 3}
+			{title: t('calendar', 'Repeating'), value: 3},
+			{title: t('calendar', 'Spreed Meeting'), value: 4}
```

Comment:
Spreed should not be used in the UI. It's "Video calls"

Reply:
This is still open, the app will be named "Talk" in 13, so it should refer to that, not spreed